packages/patternfly-4/react-core/src/components/Wizard/Wizard.tsx

```diff
@@ -414,61 +429,77 @@ export class Wizard extends React.Component<WizardProps, WizardState> {
       activeStep
     };
 
-    return (
-      isOpen && ReactDOM.createPortal(
+    if (!isOpen) {
+      return null;
+    }
+
+    const wizard = (
+        <WizardContextProvider value={context}>
+          <div {...rest}
+               className={css(
+                 styles.wizard,
+                 !this.isModal && styles.modifiers.inPage,
+                 isCompactNav && 'pf-m-compact-nav',
+                 activeStep.isFinishedStep && 'pf-m-finished',
+                 setFullWidth && styles.modifiers.fullWidth,
+                 setFullHeight && styles.modifiers.fullHeight,
+                 className)}
+               role="dialog"
```

Comment:
i think this is only applicable to modal

Reply:
fixed